Python/emscripten_signal.c

```diff
@@ -38,19 +38,17 @@ _Py_CheckEmscriptenSignals(void)
     }
 }
 
-
 #define PY_EMSCRIPTEN_SIGNAL_INTERVAL 50
 static int emscripten_signal_clock = PY_EMSCRIPTEN_SIGNAL_INTERVAL;
 
 void
 _Py_CheckEmscriptenSignalsPeriodically(void)
 {
-    if (!Py_EMSCRIPTEN_SIGNAL_HANDLING) {
-        return;
-    }
-    emscripten_signal_clock--;
     if (emscripten_signal_clock == 0) {
         emscripten_signal_clock = PY_EMSCRIPTEN_SIGNAL_INTERVAL;
         _Py_CheckEmscriptenSignals();
     }
```

Comment:
I still think we should keep the guard:
```suggestion
    if (!Py_EMSCRIPTEN_SIGNAL_HANDLING) {
        return;
    }
    if (emscripten_signal_clock == 0) {
        emscripten_signal_clock = PY_EMSCRIPTEN_SIGNAL_INTERVAL;
        _Py_CheckEmscriptenSignals();
    }
```